### ISSUE TITLE
Question about the test  in Table 3

### ISSUE THREAD

**Issue by THU-cui** [andrewowens/multisensory#19]
Did you train a special model  on short (200 ms) videos ?Because I find that the model in "/results/nets/sep/full/" can't run on video shorter than 2.135 ms.

**Comment by andrewowens** [comment]
Yes, I trained a separate model for that, which took much shorter video clips as input.
